lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -4821,12 +4842,6 @@ instance IsServerError ErrUpdateSealedTx where
                 , "the transaction body will render witnesses invalid!"
                 , "Please make sure to remove all key witnesses from the request."
                 ]
-        ErrByronTxNotSupported{} ->
-            apiError err501 NotImplemented  $ T.unwords
```

Comment:
We used to have two different ways to render "byron not supported" 🙀

I think err501 is wrong:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/501

However, perhaps 405 should be used instead of 403 on line 4324...

> If the server does recognize the method, but intentionally does not support it, the appropriate response is [405 Method Not Allowed](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405).

Reply:
Will consider changing tomorrow.

Reply:
On second look, err400 doesn't seem right either. I'm sticking with err403, which we already are using a lot.